target/linux/mvebu/base-files/lib/mvebu.sh

```diff
@@ -17,6 +17,9 @@ mvebu_board_detect() {
 	*"Marvell Armada 370 Evaluation Board")
 		name="armada-370-db"
 		;;
+	*"WD My Cloud EX2 Ultra")
```

Comment:
This reminds me of this [patch series](https://patchwork.ozlabs.org/project/openwrt/list/?series=105587) from @tmn505 where with this [patch](https://patchwork.ozlabs.org/patch/1093770/) we could start using DT based board detection so this could be dropped.

Reply:
Patch is in master and PR is adopted to it